src/renderer.zig

```diff
@@ -329,6 +378,134 @@ pub fn renderWorld(world: *World, ambientLight: Vec3f, skyColor: Vec3f, playerPo
 	gpu_performance_measuring.stopQuery();
 }
 
+pub const TransparencyMode = enum {
```

Comment:
If it's just used once in one function, then it should be private and maybe even anonymous (which would be fine if you split the function signature into multiple lines)